src/Blog/templates/list.phtml

```diff
@@ -14,7 +14,12 @@ $this->end();
 <section class="blog container">
     <div class="row align-items-start mt-5">
         <div class="col">
-            <h1><?= $this->escapeHtml($title) ?></h1>
+            <h1>
+                Blog
+                <?php if (isset($tag)) : ?>
+                    <small>Tag: <?= $this->escapeHtml($tag) ?></small>
```

Comment:
can we have just one tag?

Reply:
This is only for listing blog posts via a tag. A single blog post can have multiple tags.

https://github.com/laminas/getlaminas.org/blob/3c490f76e05ec9f2e9653f1d16ac117defa376aa/src/Blog/Handler/ListPostsHandler.php#L107-L120

https://github.com/laminas/getlaminas.org/blob/3c490f76e05ec9f2e9653f1d16ac117defa376aa/src/Blog/ConfigProvider.php#L80-L83